modules/web-console-overview.adoc

```diff
@@ -20,3 +20,8 @@ INFO Login to the console with user: kubeadmin, password: <provided>
 Use those details to log in and access the web console.
 
 For existing clusters that you did not install, you can use `oc whoami --show-console` to see the web console URL.
+
+[IMPORTANT]
+====
+The `dir` parameter specifies the assets directory, which stores the manifest files, the ISO image, and ** auth ** directory. The ** auth ** directory stores the ** kubeadmin-password ** and ** kubeconfig ** files. As a `kubeadmin` user, you can use `kubeconfig` to access the cluster with the following setting: `export KUBECONFIG=<install_directory>/auth/kubeconfig`. Note that the `kubeconfig` is specific to the generated ISO image, so if the `kubeconfig` is set and the `oc` command fails, it is possible that the system did not boot with the generated ISO image. You can use the contents of ** kubeadmin-password ** file to log into the console during the bootstrap as the `core` user to perform debugging.
+====
```

Comment:
@bfournie Can you please review?

Reply:
This looks fine but should we create a separate bug to add additional troubleshooting info as suggested by Zane in https://github.com/openshift/openshift-docs/pull/55668#discussion_r1105243059

Reply:
Yes. We can create a separate bug to address that. For now, I will get this PR merged.